update-version-and-changelog.js

```diff
@@ -0,0 +1,84 @@
+const semver = require('semver');
```

Comment:
More of a tidiness comment, but would it make sense to put this file, along with the update-google-fonts script in a `utils` folder or something along those lines?

Reply:
Yep, good idea, we can follow up that in another PR.